### PR TITLE
[CHORE] Adds languages documentation

### LINKED ISSUE #2611: [DOCUMENTATION] Add documentation explaining languages, keywords and more
**Description**
We currently don't describe the language support, dynamic keywords and the dynamic switcher. It would be nice to add a dedicated .MD file to explain and document all these features.

### PR DESCRIPTION
**Pull request by TiBiBa**
**Description**
In this PR we add a short explanation on how the languages and keyword languages work within Hedy. This feature has been deployed a few months ago, but was never properly documented. In this PR we mark a starting point of better explaining how the language and keyword language work within our project.

**Fixes**
This PR fixes #2611.

**How to test**
Not much to check as it is only an .MD file.

### CODE REVIEW

File: LANGUAGES.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## Languages
+Hedy currently supports a total of 23 languages, for which some are partly translated. 
+Alle translations are handled by Weblate, enabling contributors to contribute within an nice UI instead of a code editor.
```
Comment (Mark-Giesen) [inline LANGUAGES.md:3]: All ipv Alle?

File: LANGUAGES.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+Non-translated strings are automatically shown in English. Adding new languages is completely automated in the translation pipeline.
+You can add a new language in Weblate and start translating. After merging the language can automatically be found on the Hedy website.
+
+Hedy currently supports a total of 9 keyword languages. The keyword language is the actual programming language and the language of the Hedy commands.
+Each programming situation has only two valid keyword languages: the currently selected language and English.
+So when the user language is set to Norsk the user can use both English and Norsk keywords, even in the same program! 
+When a language does not have translated keywords English is the valid keyword language. The user set keyword language can only be English or the selected user language.
+So for example, when the profile language is set to Dutch only Dutch and English are valid keyword languages.
+
+All examples, adventures, quizzes and code snippets are shown with the selected keyword language by dynamically parsing on runtime.
```
Comment (Mark-Giesen) [inline LANGUAGES.md:13]: at runtime

**Review by Mark-Giesen — CHANGES_REQUESTED** [review]

File: LANGUAGES.md (hunk 1 above)
Comment (TiBiBa) [reply LANGUAGES.md:3]: Good catch, fixed!

File: LANGUAGES.md (hunk 2 above)
Comment (TiBiBa) [reply LANGUAGES.md:13]: Fixed!

**Review by Mark-Giesen — APPROVED** [review]

**Pull request merged**